.travis.yml

```diff
@@ -61,6 +61,7 @@ script:
   - flake8
   - autopep8 -r . --diff | tee check_autopep8
   - test ! -s check_autopep8
+  - python -Werror::DeprecationWarning -m compileall -f -q chainer chainermn examples tests
```

Comment:
Why don't you fix `docs/` and test it by `python -Werror::DeprecationWarning -m compileall -f -q .`?

Reply:
I personally prefer specifying dirs explicitly rather than `.`, because contributors may want to run it locally (after seeing Travis failure in their PR), and there may other python source files that are not intended to commit.